src/main.ts

```diff
@@ -3,7 +3,10 @@ import {server} from './server/server';
 
 (async () => {
   try {
-    const httpServer = await server(await Catalog.default());
+    const httpServer = await server(await Catalog.default(), [
+      // Only a single reconciliation service for now.
+      'https://data.netwerkdigitaalerfgoed.nl/rkd/rkdartists/sparql',
```

Comment:
Inject enabled reconciliation services into the server (so this could come from an env var or secondary catalog later) but hard-code it for now.

Reply:
Add another source: CHT.